### DISCUSSION TITLE
Consume actions and states between two different isolated store files

### DISCUSSION THREAD

**Discussion by eneskabacaoglu** [pmndrs/zustand#832] [Q&A] [answered]
I am creating files for my each store like so : 

Stores
|__ userStore.js
|__ userNotesStore.js

Lets say, I would like to use an action from userStore.js in userNotesStore.js. When I import userStore.js into userNotesStore.js, I get error. But that's the only way I know for consuming any store's actions or states. (Like in a component for example) 

My questions is how can I use actions and states **between two isolated store files** ? 

Thank you for your help !

**Comment by dai-shi** [comment, marked as answer]
While I would generally recommend considering combining into one single store in such use cases, what you are probably looking for is `useStore.getState()`.

**Reply by eneskabacaoglu** [reply]
Thank you @dai-shi
This is exactly what I was looking for and worked as expected. 

I totally agree with your suggestion. Just to make it much more clear, I have used a dummy scenario. So, however combining the stores sounds like a perfect solution for this one, in my real-life scenario, I needed to call a state in a different store. So I had to look for an option. 

Thanks for your amazing effort and support. Cheers
